experimental/ExtendTeamsforM365/client/pages/categoryDetail.html

```diff
@@ -6,6 +6,17 @@
     <title>Northwind Product Category Detail</title>
     <link rel="stylesheet" href="/northwind.css" />
     <link rel="icon" href="data:;base64,="> <!-- Suppress favicon error -->
+    <style>               
```

Comment:
Move this to css file.

Reply:
Changed and shifted to northwind-teams.css